packages/office-addin-debugging/package.json

```diff
@@ -29,7 +29,8 @@
     "office-addin-dev-settings": "^1.8.19",
     "office-addin-manifest": "^1.5.19",
     "office-addin-node-debugger": "^0.4.41",
-    "office-addin-usage-data": "^1.2.0"
+    "office-addin-usage-data": "^1.2.0",
+    "read-package-json-fast": "^2.0.2"
```

Comment:
Let's not add this to every package. You should have the functionality in one package that can then be used by the others.

Reply:
I don't think I got it. I only added this one new package to the ```office-addin-debugging``` package and not any of the others.

Reply:
This won't be specific to debugging. This should be usable for other places either now or in the future.

Reply:
Being in ```office-addin-cli``` with some default functions there fixes this problem right?